<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTrainModal</class>
 <widget class="QWidget" name="AddTrainModal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font-weight: 600; font-size: 10pt; color: black;</string>
     </property>
     <property name="text">
      <string>Поезд</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,0">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="formLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item row="3" column="1">
          <layout class="QVBoxLayout" name="inputGroup_11">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="secondClassLabel">
             <property name="text">
              <string>Цена 2 класса руб.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="secondClassInput">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>20000</number>
             </property>
             <property name="singleStep">
              <number>100</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <layout class="QVBoxLayout" name="inputGroup_9">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="railcarAmountLabel">
             <property name="text">
              <string>Количество вагонов</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="railcarAmountInput"/>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QVBoxLayout" name="inputGroup_8">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="railcarCapLabel">
             <property name="text">
              <string>Вместимость</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="railcarCapInput">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>200</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="inputGroup_6">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="nameLabel">
             <property name="text">
              <string>Название</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameInput">
             <property name="echoMode">
              <enum>QLineEdit::Normal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QVBoxLayout" name="inputGroup_10">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="firstClassLabel">
             <property name="text">
              <string>Цена 1 класса руб.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="firstClassInput">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>20000</number>
             </property>
             <property name="singleStep">
              <number>100</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="inputGroup_7">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="trainTypeLabel">
             <property name="text">
              <string>Тип поезда</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="trainTypeComboBox">
             <item>
              <property name="text">
               <string>пассажирский</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>скорый-пассажирский</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>скоростной-пассажирский</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0" colspan="2">
          <layout class="QVBoxLayout" name="inputGroup_12">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="routeLabel">
             <property name="text">
              <string>Маршрут</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="routeComboBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
